fitbenchmarking/results_processing/performance_profiler.py

```diff
@@ -1,14 +1,14 @@
 """
 Set up performance profiles for both accuracy and runtime tables
 """
-from collections import OrderedDict
 import os
+from collections import OrderedDict
 from textwrap import wrap
+import numpy as np
+import matplotlib.pyplot as plt
 
 import matplotlib
 matplotlib.use('Agg')
-import matplotlib.pyplot as plt
```

Comment:
I think the matplotlib.pyplot import has to come after the Agg line. I can't remember why -- it should probably have been commented here!

Having looked it up, it appears to be due to needing to save to file as described in [this answer](https://stackoverflow.com/a/20249447). There might also be more info in the git history.